RecoTauTag/RecoTau/python/tools/runTauIdMVA.py

```diff
@@ -728,10 +728,18 @@ def runTauID(self):
                 disable_dxy_pca          = cms.bool(True),
                 is_online                = cms.bool(False)
             ))
+            _deepTauProducer = getattr(self.process,_deepTauName+self.postfix)
+
+            # only implemented Sonic DeepTau for deepTau2017v2p1
+            # if/when implementing for other versions, reduce duplication of setattr(...) lines above
+            from RecoTauTag.RecoTau.deepTauSonicProducer_cff import deepTauSonicProducer as _deepTauSonicProducer
```

Comment:
I think that this cff in not mandatory (see other comment). Please import default auto-generated cfi and clone it with desired modifications.